e2eRunner/src/main/scala/za/co/absa/hermes/e2eRunner/plugins/BashPlugin.scala

```diff
@@ -53,13 +53,17 @@ class BashPlugin extends Plugin {
 
   override def performAction(testDefinition: TestDefinition, actualOrder: Int): BashJobsResult = {
     def runBashCmd(bashCmd: String): String = {
-      (s"echo $bashCmd" #| "bash").!!
+      if (System.getProperty("os.name").toLowerCase().contains("windows")) {
+        (s"cmd /C ${bashCmd.replace("'", "\"")}").!!
+      } else {
+        (s"echo $bashCmd" #| "bash").!!
```

Comment:
Out of curiosity what the `.!!` at the end means/does?

Reply:
!!: blocks until all external commands exit, and returns a String with the output generated.
[Source](https://www.scala-lang.org/api/2.13.4/scala/sys/process/ProcessBuilder.html).